### PR TITLE
Fix/trends rout

### PR DESCRIPTION
**Pull request by KirillSerg**
fix trends link

### CODE REVIEW

**Review by DavidKorystov — APPROVED** [review]

**Review by DmytroTikhostup — APPROVED** [review]
Nice

File: src/components/RightPanel.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ export const RightPanel = () => {
       <StickySearch>
         <AsyncSearch />
       </StickySearch>
-      {!window.location.pathname.includes("/trends") && <Trends />}
+      {!window.location.hash.includes("/trends") && <Trends />}
```
Comment (alexandrtovmach) [inline src/components/RightPanel.tsx]: use react-router-dom hook to access to hash instead of window.location

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

File: src/containers/Trends.tsx [hunk 2]
```diff
@@ -1,16 +1,9 @@
...
 import { TrendItem } from "../components/TrendItem";
 import { MockTrend } from "../types/mock-api-types";
-
-const StyledLink = styled.a`
-  transition: 0.3s;
-  :hover {
-    background-color: rgba(0, 0, 0, 0.03);
-  }
-`;
+import { LinkWithLanguageQueryParam } from "./LinkWithLanguageQueryParam";
```
Comment (vasyagladush) [inline src/containers/Trends.tsx:7]: If you want to style this link you can write: 

```
const StyledLink = styled(LinkWithLanguageQueryParam)`
      // stylisation
 `;
```

**Review by vasyagladush — APPROVED** [review]

File: src/App.tsx [hunk 3]
```diff
@@ -63,7 +63,7 @@ const App: React.FunctionComponent = () => {
         <Route path="/authorization" element={<AuthorizationPage />} />
         <Route path="/followers" element={<Followers />} />
         <Route path="/following" element={<Following />} />
-        <Route path="/trends" element={<Trends />} />
+        <Route path="/:trends" element={<Trends />} />
```
Comment (alexandrtovmach) [inline src/App.tsx]: what? no it's not what I've asked

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 3 above)
Comment (alexandrtovmach) [reply src/App.tsx]: you need to use hook to get path, not param

Comment (KirillSerg) [reply src/App.tsx]: sorry. Changed

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**